testsuite/fixtures/toolchain_index/gn/gnat_external/gnat_external-external.toml

```diff
@@ -7,6 +7,6 @@ maintainers-logins = ["mosteo"]
 [[external]]
 kind = "version-output"
 # We look for make instead that should be always installed.
```

Comment:
This comment is obsolete now

Reply:
I missed this comment. Maybe I shouldn't change this.

Reply:
I just checked and this change is not needed. So I will revert.